src/pages/rsvp/showandtell.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Layout, Section, SectionContent } from '../../components/layout'
+import * as S from '../../page-components/projects/styles'
+import { Input } from '../../components/inputs'
+import { Button } from '../../components/buttons'
```

Comment:
Úprava všech možných relativních importů na absolutní (všechny složky v `src`).

Reply:
Jak nastavim IntelliJ aby tohle delal za me? Dava mi na vyber  "relativni" vs "absolutni", kde je to i s prefixem src/, coz nefunguje.